python/dgl/distributed/rpc.py

```diff
@@ -422,16 +706,16 @@ def remote_call(target_and_requests, timeout=0):
         # recv response
         msg = recv_rpc_message(timeout)
         num_res -= 1
-        if msg is None:
-            continue
```

Comment:
this check is invalid?

Reply:
recv_rpc_message will also return a msg. If no message is coming, the recv() will just be blocked.

Reply:
what if it times out?

Reply:
For now, I didn't handle times out. The recv will always listen to the port until recv a msg for shutdown.

Reply:
but in the future, you need to deal with this case, right? maybe you can keep it so you don't need to add it back?